src/main.js

```diff
@@ -49,10 +49,26 @@ window.onload = (event) => {
   coverTitle.innerHTML = currentCover.title;
   coverDescriptor1.innerHTML = currentCover.tagline1;
   coverDescriptor2.innerHTML = currentCover.tagline2;
+
+  var editableCoverFields = fullCover.querySelectorAll('img.cover-image, h2.cover-title, h3.tagline');
+  editableCoverFields.forEach((element) => {
+    element.addEventListener('click', function () {
+      getNewRandomCoverItem(element);
+      assignNewCoverID();
+    });
+    element.addEventListener('dblclick', function () {
+      manuallyEditCover(element);
+      assignNewCoverID();
+    });
+  });
```

Comment:
It looks like our `getRandomCover()` function is not retaining these event listeners assigned `onload`.  So we'll need to look into how to add those back after that function is called.